app/styles/studio.css

```diff
@@ -517,10 +517,16 @@ ANALOGIC SCHEME
     min-height: 2em !important;
 }
 
+.textareavalues {
+    min-height: 1em !important;
+    min-width: unset;
```

Comment:
Why did you do ' min-width: unset;' ?

Reply:
otherwise it inherits from min-width: 90% which does not align with other fields in the modal form

Reply:
but maybe there is a better approach than unset

Reply:
Another solution would be to look at the CSS of the other fields. But this seems OK too.

Reply:
ok, I will leave it like this then.